unicodetools/data/security/dev/data/source/removals.txt

```diff
@@ -846,3 +846,21 @@ AB68; Technical
 # UTC-179-C39: Change the Identifier_Type of A9CF to Limited_Use Uncommon_Use, removing Exclusion.
 # (A code fix removed Exclusion from Limited_Use characters, but adding Uncommon_Use needs an override.)
 A9CF; Limited_Use Uncommon_Use
+
+# Unicode 17
+09FF; Technical # L2/22-268R
+0B53..0B54; Uncommon_Use # L2/24-106R
+0C5C; Obsolete # L2/22-006
+0CDC; Obsolete # L2/22-006
+1ACF..1ADD; Technical # L2/24-105, L2/23-188, L2/23-208, L2/23-206
+1AE0..1AEA; Technical # L2/24-080
+A7CE..A7CF; Uncommon_Use # L2/23-219
+A7D2; Obsolete # L2/23-135
+A7D4; Obsolete # L2/23-135
+10EC5; Technical # L2/22-281R
+10EC6; Technical # L2/23-121
+10EC7; Uncommon_Use # L2/24-055
+10EFB; Technical # L2/22-281R
+16FF2..16FF6; Technical # L2/24-071R3
```

Comment:
Sorry, I wasn't very clear on 16FF2..16FF6. I said "add L2/24-071R3". What I meant was "keep L2/23-284 and add L2/24-071R3". So now L2/23-284 needs to be added back.

Reply:
Sorry I misinterpreted your comment :-) Added back in https://github.com/unicode-org/unicodetools/pull/1062/commits/d018a2caf051082527617a54bc00949c665b4a35